doc/src/manual/getting-started.md

```diff
@@ -14,7 +14,7 @@ io = IOBuffer()
 Base.banner(io)
 banner = String(take!(io))
 import Markdown
-Markdown.parse("```\n\$ julia\n\n$(banner)\njulia> 1 + 2\n3\n\njulia> ans\n3\n```")
```

Comment:
I think it's useful to keep this in the getting started docs, since people may not be familiar with this.

Reply:
(If you want to make it more explicit, you can do something like)
```
julia> ans # `ans` retrieves the previous REPL answer
3
```